nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyControllerV1.java

```diff
@@ -40,6 +41,7 @@
 
   public void attach(final Router.Builder builder) {
     createRoute(builder, ConanRoutes.downloadUrlsProxy(), DOWNLOAD_URL, proxyHandler);
+    createRoute(builder, ConanRoutes.digest(), DIGEST, proxyHandler);
     createRoute(builder, ConanRoutes.conanManifestProxy(), CONAN_MANIFEST, proxyHandler);
```

Comment:
Please, rename to ConanRoutes.digest() -> ConanRoutes.digestProxy()

Reply:
Fixed 7e9eed00a7293a6b57a745c69ea24ddfc1a19f61